shared/AppInsightsCommon/src/Interfaces/IConfig.ts

```diff
@@ -390,6 +390,19 @@ export interface IConfig {
      */
     userOverrideEndpointUrl?: string;
 
+    /**
+     * [Optional] Specifies the Cross-Origin Resource Policy (CORP) for the endpoint.
+     * This value is included in the response header as `Cross-Origin-Resource-Policy`,
+     * which helps control how resources can be shared across different origins.
+     *
+     * Possible values:
+     * - `same-site`: Allows access only from the same site.
+     * - `same-origin`: Allows access only from the same origin (protocol, host, and port).
+     * - `cross-origin`: Allows access from any origin.
+     *
+     * @since 3.3.3
+     */
+    crossOriginResourcePolicy?: string;
```

Comment:
Lets keep this in just the `ISenderConfig` so if you want to set it you set via the Sender Config as this is not a common config, so we should not have it for "everyone" (at least in TypeScript -- technically, the code will also pick it up from the root, but we are going to say -- don't configure that way use the sender config approach and we will only test that way as the "supported" method.